firebase-installations/src/main/java/com/google/firebase/installations/remote/FirebaseInstallationServiceClient.java

```diff
@@ -91,16 +91,16 @@ public InstallationResponse createFirebaseInstallation(
           return readCreateResponse(httpsURLConnection);
         case 401:
           throw new FirebaseInstallationServiceException(
-              UNAUTHORIZED_ERROR_MESSAGE, FirebaseInstallationServiceException.Code.UNAUTHORIZED);
+              UNAUTHORIZED_ERROR_MESSAGE, FirebaseInstallationServiceException.Status.UNAUTHORIZED);
         default:
```

Comment:
Can this ever happen?
How is this a Server-Error?
Seems that this is a client error for not catching all possible error codes.
I think, we should give more information to the developer what is happening here.

Reply:
Added a TODO in FirebaseInstallationsException class. I will send out another PR for exception handling changes.